lib/utils/bedroomCalculator.ts

```diff
@@ -0,0 +1,60 @@
+function calculateBedrooms(...people: Array<any>) {
```

Comment:
We need to type the people array properly. `people` is an array, but it's an array of tuples where the first index is an age and the second is a gender. The correct type to use is `[number, string][]`